pkg/processor/processor.go

```diff
@@ -45,6 +46,8 @@ func (s *OperationProcessor) Resolve(uniqueSuffix string) (document.Document, er
 		return nil, err
 	}
 
+	sortOperations(ops)
```

Comment:
Store is supposed to return sorted operations (in chronological order). I guess it doesn't hurt to do it here too.